Makefile

```
@@ -55,3 +55,9 @@ uninstall-pre-commit:
 
 print-python-env:
 	@./tools/print_env.sh
+
+auto-sign-off:
```

I like a shorter name for this, but it's entirely optional.

```suggestion
sign-off:
```